client/src/partials/locations/locations.html

```diff
@@ -9,32 +9,30 @@
 
 <!-- Menu to manage the various locations -->
 <div class="flex-util">
-  <a href="#/locations/village/" id="village" class="btn btn-default btn-sm" style="margin: 0 2px;">
-    <span class="glyphicon glyphicon-home"></span> {{ 'LOCATION.MANAGE_VILLAGE' | translate }}
+  <a ui-sref="locationsVillage" id="village" class="btn btn-default btn-sm" style="margin: 0 2px;" href>
```

Comment:
Ah, didn't really pay attention to the class.  I'll make the change.  Good catch.